app/controllers/application_controller.rb

```diff
@@ -1,2 +1,25 @@
-class ApplicationController < ActionController::API
+class ApplicationController < ActionController::Base
+  protect_from_forgery unless: -> { request.format.json? }
+
+  before_action :require_user
+
+  helper_method :logged_in?, :current_user
+
+  def logged_in?
+    cookies.encrypted[:current_user_id].present?
+  end
+
+  def current_user
+    if cookies.encrypted[:current_user_id].present?
+      begin
+        User.find(cookies.encrypted[:current_user_id])
```

Comment:
You can use `find_by(id: ...)`, which returns `nil` if the user is not found; that way you could cut a few lines.